aio/package.json

```
@@ -56,7 +56,7 @@
     "docs-test": "node tools/transforms/test.js",
     "redirects-test": "node tests/deployment/unit/test",
     "firebase-utils-test": "node tools/firebase-test-utils/test",
-    "tools-lint": "tslint --config \"tools/tslint.json\" --project \"tools/firebase-test-utils\" && eslint tools/ng-packages-installer",
+    "tools-lint": "eslint \"tools/firebase-test-utils\" && eslint tools/ng-packages-installer",
```

Nit: Let's keep things simple :wink:

```suggestion
    "tools-lint": "eslint tools/firebase-test-utils && eslint tools/ng-packages-installer",
```